### PR TITLE
Fixes CloudBank apps enabling observability

### LINKED ISSUE #653: Grafana is not showing instances to view metrics
Stack: v3
Grafana is not showing instances for services deployed via oractl.

In this scenario, cloudbank jars were successfully deployed using oractl. Grafana is not showing any instance to view the metrics

image

Note:
Also deployed slowapp using `kubectl apply -f <yaml_file>` and grafana shows metrics for slowapp instance as expected.

### PR DESCRIPTION
**Pull request by pasimoes**
As described in issue #653, the metrics from CloudBank services were not showing in Grafana Dashboard. This PR resolves #653 

Fixes:
-- Make services discoverable by adding Eureka Client
-- Address conflict between Eureka Client and Jersey 2 (jersey starter)
-- Configure the management tag in the application.yaml

### CODE REVIEW

File: cloudbank-v2/spring-apps/creditscore/pom.xml [hunk 1]
```diff
@@ -4,65 +4,51 @@
...
-                <artifactId>spring-cloud-dependencies</artifactId>
-                <version>${spring-cloud.version}</version>
-                <type>pom</type>
-                <scope>import</scope>
-            </dependency>
-        </dependencies>
-    </dependencyManagement>
+    <description>Creditscore Application</description>
+
+<!--    <dependencies> -->
```
Comment (markxnelson) [inline cloudbank-v2/spring-apps/creditscore/pom.xml:19]: don't leave blocks of commented out code - just delete it

Comment (markxnelson) [reply cloudbank-v2/spring-apps/creditscore/pom.xml:19]: applies to other places in this same pr too

**Review by markxnelson — CHANGES_REQUESTED** [review]
don't leave huge blocks of commented out code.
in general, i am ok with merging this provided two things are true:
1. both andy and paulo have tested this and confirmed it works
2. there is a pr ready to make the corresponding updates to the cloudbank live lab as well, and that has also been tested and is working

File: cloudbank-v2/spring-apps/account/pom.xml [hunk 2]
```diff
@@ -4,90 +4,101 @@
...
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-starter-netflix-eureka-client</artifactId>
+			<exclusions>
+				<exclusion>
+					<groupId>javax.ws.rs</groupId>
+					<artifactId>jsr311-api</artifactId>
+				</exclusion>
+				<exclusion>
+                    <groupId>javax.servlet</groupId>
```
Comment (markxnelson) [inline cloudbank-v2/spring-apps/account/pom.xml:42]: can we fix the formatting?

**Review by markxnelson — CHANGES_REQUESTED** [review]
some formatting issues and missing notices

File: cloudbank-v2/spring-apps/account/pom.xml (hunk 2 above)
Comment (andytael) [reply cloudbank-v2/spring-apps/account/pom.xml:42]: Formatting looks good in VS COde and IntelliJ, not sure how to fix @markxnelson

File: cloudbank-v2/spring-apps/transfer/src/main/java/com/example/transfer/servlet/filter/SpringMvcPrefixEnforcerFilter.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
+package com.example.transfer.servlet.filter;
```
[the remark this answers is not in the document]
Comment (andytael) [reply cloudbank-v2/spring-apps/transfer/src/main/java/com/example/transfer/servlet/filter/SpringMvcPrefixEnforcerFilter.java:4]: @mnelson Fixed

**Review by markxnelson — APPROVED** [review]

**Pull request merged**